rentals.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'person'
+require_relative 'book'
+
+class Rentals
+  attr_accessor :date
+  attr_reader :person, :book
+
```

Comment:
- [ ] Please `Getters and Setters for @book and @person instance variables should be present`